### PR TITLE
Add more coverage for partition table entity

### PR DESCRIPTION
**Pull request by oshtaier**
```
nosetests tests/foreman/ui/test_partitiontable.py
..............
----------------------------------------------------------------------
Ran 14 tests in 888.039s

OK
```
```
nosetests tests/foreman/ui/test_organization.py -m test_positive_remove_ptable
.
----------------------------------------------------------------------
Ran 1 test in 367.418s

OK
```

### CODE REVIEW

**Comment by oshtaier** [comment]
Part of #3332. Still need to try to automate 'History' sub-tab

File: robottelo/ui/partitiontable.py [hunk 1]
```diff
@@ -17,27 +17,34 @@ def _search_locator(self):
         """Specify locator for Partition Table entity search procedure"""
         return locators['ptable.ptable_name']
 
-    def _configure_partition_table(self, os_family=None):
-        """Configures the os family of partition table."""
+    def _configure_partition_table(
+            self, audit_comment=None, default=None, os_family=None,
```
Comment (abalakh) [inline robottelo/ui/partitiontable.py]: nitpick: no new line needed here

File: tests/foreman/ui/test_organization.py [hunk 2]
```diff
@@ -724,6 +724,41 @@ def test_positive_remove_template(self):
...
+                    make_org(
+                        session, org_name=org_name, ptables=[ptable_name])
+                    self.org.click(self.org.search(org_name))
+                    self.org.click(tab_locators['context.tab_ptable'])
+                    element = self.org.wait_until_element(
+                        (strategy1, value1 % ptable_name))
+                    # Item is listed in 'Selected Items' list and not
+                    # 'All Items' list.
+                    self.assertIsNotNone(element)
+                    self.org.update(org_name, ptables=[ptable_name])
```
Comment (abalakh) [inline tests/foreman/ui/test_organization.py]: I don't get it. the org was created with ptable assigned and now you're updating it with ptable again? How ptable appears unassigned then? Or it is intended behavior and to deselect ptable you need to 'select' it again?

Comment (oshtaier) [inline tests/foreman/ui/test_organization.py]: Yeap
`def update(...select=False...)`
Anyway, that module require total refactoring.

**Comment by abalakh** [comment]
ACK pending question

File: tests/foreman/ui/test_organization.py (hunk 2 above)
Comment (abalakh) [inline tests/foreman/ui/test_organization.py]: d'oh. yeah, that module definitely needs refactoring. thanks for clarification

**Comment by rochacbruno** [comment]
ACK: pending rebase + new testimony tags

File: tests/foreman/ui/test_organization.py [hunk 3]
```diff
@@ -724,6 +724,41 @@ def test_positive_remove_template(self):
 
     @run_only_on('sat')
     @tier2
+    def test_positive_remove_ptable(self):
```
Comment (elyezer) [inline tests/foreman/ui/test_organization.py]: This test will require docstring update.

File: tests/foreman/ui/test_partitiontable.py [hunk 4]
```diff
@@ -94,6 +67,167 @@ def test_positive_create_with_name(self):
 
     @run_only_on('sat')
     @tier1
+    def test_positive_create_with_snippet(self):
```
Comment (elyezer) [inline tests/foreman/ui/test_partitiontable.py]: This test will require docstring update.

File: tests/foreman/ui/test_partitiontable.py [hunk 5]
```diff
@@ -94,6 +67,167 @@ def test_positive_create_with_name(self):
...
+                session,
+                name=name,
+                template_path=PARTITION_SCRIPT_DATA_FILE,
+                snippet=True,
+            )
+            self.assertIsNotNone(self.partitiontable.search(name))
+
+    @run_only_on('sat')
+    @tier1
+    def test_positive_create_with_audit_comment(self):
```
Comment (elyezer) [inline tests/foreman/ui/test_partitiontable.py]: This test will require docstring update

File: tests/foreman/ui/test_partitiontable.py [hunk 6]
```diff
@@ -94,6 +67,167 @@ def test_positive_create_with_name(self):
...
+                        session,
+                        name=name,
+                        template_path=PARTITION_SCRIPT_DATA_FILE,
+                        audit_comment=comment_text,
+                    )
+                    self.assertIsNotNone(self.partitiontable.search(name))
+
+    @run_only_on('sat')
+    @tier2
+    def test_positive_create_default_for_organization(self):
```
Comment (elyezer) [inline tests/foreman/ui/test_partitiontable.py]: This test will require docstring update

File: tests/foreman/ui/test_partitiontable.py [hunk 7]
```diff
@@ -94,6 +67,167 @@ def test_positive_create_with_name(self):
...
+            session.nav.click(self.org.search(org_name))
+            session.nav.click(tab_locators['context.tab_ptable'])
+            # Item is listed in 'Selected Items' list and not 'All Items' list.
+            self.assertIsNotNone(
+                session.nav.wait_until_element((strategy1, value1 % name))
+            )
+
+    @run_only_on('sat')
+    @tier2
+    def test_positive_create_non_default_for_organization(self):
```
Comment (elyezer) [inline tests/foreman/ui/test_partitiontable.py]: This test will require docstring update

File: tests/foreman/ui/test_partitiontable.py [hunk 8]
```diff
@@ -94,6 +67,167 @@ def test_positive_create_with_name(self):
...
+            session.nav.click(self.org.search(org_name))
+            session.nav.click(tab_locators['context.tab_ptable'])
+            # Item is listed in 'All Items' list and not Selected Items' list.
+            self.assertIsNotNone(
+                session.nav.wait_until_element((strategy1, value1 % name))
+            )
+
+    @run_only_on('sat')
+    @tier2
+    def test_positive_create_default_for_location(self):
```
Comment (elyezer) [inline tests/foreman/ui/test_partitiontable.py]: This test will require docstring update

File: tests/foreman/ui/test_partitiontable.py [hunk 9]
```diff
@@ -94,6 +67,167 @@ def test_positive_create_with_name(self):
...
+            session.nav.click(self.location.search(loc_name))
+            session.nav.click(tab_locators['context.tab_ptable'])
+            # Item is listed in 'Selected Items' list and not 'All Items' list.
+            self.assertIsNotNone(
+                session.nav.wait_until_element((strategy1, value1 % name))
+            )
+
+    @run_only_on('sat')
+    @tier2
+    def test_positive_create_non_default_for_location(self):
```
Comment (elyezer) [inline tests/foreman/ui/test_partitiontable.py]: This test will require docstring update

File: tests/foreman/ui/test_partitiontable.py [hunk 10]
```diff
@@ -205,20 +332,39 @@ def test_positive_update(self):
...
+                        new_template_path=PARTITION_SCRIPT_DATA_FILE,
+                        new_os_family='Red Hat',
+                    )
+                    self.assertIsNotNone(self.partitiontable.search(new_name))
+                    name = new_name  # for next iteration
+
+    @run_only_on('sat')
+    @tier1
+    def test_negative_update_name(self):
+        """Update invalid name in partition table
```
Comment (elyezer) [inline tests/foreman/ui/test_partitiontable.py]: This test will require docstring update

**Comment by elyezer** [comment]
In order to update the docstring, if the module already have the global tags on its docstring, then on each test feature token should be dropped and the id should be generated. To generate the ID use uuid module like `import uuid; print(str(uuid.uuid4()))`.

**Comment by oshtaier** [comment]
rebased

**Comment by abalakh** [comment]
ACK

**Pull request merged**